ProtocolHandler.pm

```diff
@@ -250,10 +283,7 @@ sub getMetadataFor {
 
 		if ($url) {
 			my $bitrate = int($song->bitrate ? ($song->bitrate / 1024) : 850) . 'k VBR FLAC';
-
-			if (main::INFOLOG && $log->is_info) {
```

Comment:
I believe displaying this might be more confusing to the majority of users who don't care about how many tracks there are in a chunk. Please leave that condition in. Thanks!